crates/bevy_pbr/src/material.rs

```diff
@@ -3,12 +3,22 @@ use bevy_reflect::TypeUuid;
 use bevy_render::{color::Color, renderer::RenderResources, shader::ShaderDefs, texture::Texture};
 
 /// A material with "standard" properties used in PBR lighting
+/// Standard property values with pictures here https://google.github.io/filament/Material%20Properties.pdf
 #[derive(Debug, RenderResources, ShaderDefs, TypeUuid)]
 #[uuid = "dace545e-4bc6-4595-a79d-c224fc694975"]
 pub struct StandardMaterial {
-    pub albedo: Color,
+    /// Doubles as diffuse albedo for non-metallic, specular for metallic and a mix for everything in between
+    pub base_color_factor: Color,
     #[shader_def]
-    pub albedo_texture: Option<Handle<Texture>>,
+    pub base_color_texture: Option<Handle<Texture>>,
+    /// Linear perceptual roughness, clamped to [0.089,1] in the shader
+    /// Defaults to minimum of 0.089
+    pub roughness_factor: f32,
+    /// Range [0,1] from dielectric to pure metallic
```

Comment:
very small nit: range should be [0.0,1.0]

Reply:
Done.